seed.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/env bash
```

Comment:
Had to replace the shebang to `#!/usr/bin/env bash` in macOS. Don't know if that should be considered an issue or not

Reply:
MacOS moment, will change that.